src/lib/components/processing/ProcessContext.svelte

```diff
@@ -68,17 +70,36 @@ This makes the state of those processes available via context.
 </script>
 
 <script lang="ts">
+  // keep track of processed documents
+  let started: number[] = [];
+
+  $: currentIds = new Set($documents.map((d) => d.doc_id));
+
   // stores we need deeper in the component tree, available via context
   setContext<ProcessContext>("processing", { addons, documents, load });
 
-  onMount(() => {
-    load();
+  onMount(async () => {
+    await load();
+    started = $documents.map((d) => d.doc_id);
```

Comment:
Should we poll the load endpoint after mounting? Otherwise, we may not be updating `started` after kicking off a new process. Right now, the only place `load` is called outside of mount is in the submit handler of the `Reprocess` form.

Reply:
I just added it to the `ConfirmRedaction` form, too.